### DISCUSSION TITLE
Question about Revanced Manager

### DISCUSSION THREAD

**Discussion by ghost** [ReVanced/revanced-discussions#994] [Question] [answered]
Is there a list somewhere that shows all the apps that Revanced Manager can patch?

**Comment by KobeW50** [comment, marked as answer]
It shows the list when you go to Patcher > Select an application.

You can also find the list on https://revanced.app/patches

**Comment by oSumAtrIX** [comment]
Yes, its in the app selection screen of ReVanced Manager.
